curiouslearning/assessment-survey-js
# Aj-117(Fixed West African English Firebase reporting issue in both Sight Words and Letter Sounds!)

…ght Words and Letter Sounds!)

 I have added fixes for West African English Firebase reporting issue in both Sight Words and Letter Sounds!

# Changes
- Added Changes for west-african-english assessment not able to properly log lang parameter on firebase!
- Along With that I have added condition for this language as well since the url is different for letter and sight-words assessment and we want them to be same!

# How to test
- Use extension Google Analytics Debugger and open the url in the console you can see the data that is being logged!

Ref: [JIRA-CODE](https://curiouslearning.atlassian.net/browse/JIRA-CODE)
